### DISCUSSION TITLE
The Frictionless Blog should have an RSS feed

### DISCUSSION THREAD

**Discussion by augusto-herrmann** [frictionlessdata/frictionlessdata.io#603] [Ideas]
Without an RSS feed, I cannot get notified when a new post is posted on the [Frictionless Blog](https://frictionlessdata.io/blog/).

Yes, I still use RSS aggregators to read blogs today. RSS is still alive and well. The issue is that blogs that have moved to walled gardens from tech giants, but RSS is still very much used in podcasts, for example.

**Comment by aborruso** [comment]
I totally agree with you

**Comment by rufuspollock** [comment]
Believe this is relatively easy to do https://github.com/webmasterish/vuepress-plugin-feed PR's welcome!
